test/parallel/test-buffer-compare-offset.js

```diff
@@ -57,7 +57,7 @@ assert.strictEqual(1, a.compare(b, Infinity, -Infinity));
 // zero length target because default for targetEnd <= targetSource
 assert.strictEqual(1, a.compare(b, '0xff'));
 
-const oor = /out of range index/;
+const oor = /^RangeError(?:\[ERR_INDEX_OUT_OF_RANGE\])?: Index out of range$/;
```

Comment:
This line can be replaced with:

`const oor = common.expectsError('ERR_INDEX_OUT_OF_RANGE');`

Will need to change the `require('../common');` to `const common = require('../common');` too.